include/SFML/System/Clock.hpp

```diff
@@ -101,14 +100,47 @@ class SFML_SYSTEM_API Clock
     /// \brief Get the elapsed time
     ///
     /// This function returns the time elapsed since the last call
-    /// to restart() (or the construction of the instance if restart()
-    /// has not been called).
+    /// to restart/reset() (or the construction of the instance
+    /// if restart/reset() has not been called).
     ///
     /// \return Time elapsed
     ///
     ////////////////////////////////////////////////////////////
     Time getElapsedTime() const;
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Set the clock state
+    ///
+    /// This function starts the clock if running is true
+    /// and pauses it if running is false.
+    ///
+    /// \param running True to start, false to pause
+    ///
+    /// \return True if operation was successful, false otherwise
```

Comment:
Thanks for pointing out docs error.

Reply:
Regarding API changes. Changing `pause()` into `stop()` is okay. Taking return values from `restart/reset()` as form of simplifying API since we manage two states now - okay. Though I would not consider deleting `restart()` method.